R/commas_linter.R

```diff
@@ -24,12 +24,16 @@ commas_linter <- function() {
     before_lints <- xml_nodes_to_lints(
       xml2::xml_find_all(xml, xpath_before),
       source_expression = source_expression,
-      lint_message = "Commas should never have a space before."
+      lint_message = "Commas should never have a space before.",
+      range_start_xpath = "number(./preceding-sibling::*[1]/@col2 + 1)",
```

Comment:
let's have a practice of commenting what the xpath is accomplishing in (non-trivial) non-standard cases?

Reply:
SGTM